MiniApp/Classes/Display/RealMiniAppView.swift

```diff
@@ -1,15 +1,18 @@
 import WebKit
 
-public class RealMiniAppView: UIView, MiniAppDisplayProtocol {
-
-    private var webView: WKWebView
+public class RealMiniAppView: UIView, MiniAppDisplayProtocol, MiniAppCallbackProtocol {
+    internal var webView: WKWebView
+    private var messageInterface: MiniAppMessageProtocol
```

Comment:
shouldn't we use delegate instead of interface ?

Reply:
I guess you are right, I need to change this and also I think we need to have a different name altogether which confuses us with Mini App (Web App) interface name as well.

Reply:
Done